src/domain/individualHarbor/property/Property.tsx

```diff
@@ -1,11 +1,33 @@
 import React from 'react';
+import classNames from 'classnames';
 
 import Text from '../../../common/text/Text';
 import styles from './property.module.scss';
 
+export interface IconProps {
+  disabled?: boolean;
+  outlined?: boolean;
+  width?: string;
+  height?: string;
+  size?: 'small' | 'standard' | 'large';
+  color?: 'standard' | 'brand' | 'critical' | 'secondary' | 'info';
+}
+
+const IconOutline: React.SFC<IconProps> = ({ children, disabled }) => {
+  return (
+    <div
+      className={classNames(styles.icon, styles.outlined, {
+        [styles.disabled]: disabled,
+      })}
+    >
+      {children}
+    </div>
+  );
+};
+
```

Comment:
Please, extract this component to its own module.

Reply:
done